hq/app/AppComponents.scala

```diff
@@ -99,7 +104,12 @@ class AppComponents(context: Context)
   private val s3Clients = AWS.s3Clients(configuration)
   private val iamClients = AWS.iamClients(configuration, availableRegions)
   private val efsClients = AWS.efsClients(configuration, availableRegions)
-  private val snsClients = AWS.snsClients(configuration)
+  val credsProvider = new AWSCredentialsProviderChain(DefaultAWSCredentialsProviderChain.getInstance(), new ProfileCredentialsProvider("security"))
```

Comment:
could we call this `securityCredentialsProvider` just as there's more than one credentials provider in this app